### ISSUE TITLE
outrageous wheel location

### ISSUE THREAD

**Issue by stephengold** [stephengold/Minie#14]
Following up on https://github.com/stephengold/jme-vehicles/issues/1, need to understand why `VehicleWheel.getWheelLocation()` set `wheelWorldLocation.y` to 1.6931809E27

**Comment by stephengold** [comment]
`printf()`s added in `btRaycastVehicle::updateWheelTransform()`

The suspension length is occasionally invalid:
```text
hardPoint = -92.849998 1.015596 673.400024
wheelDirection = -0.000000 -1.000000 0.000000
suspensionLength = -663963820797107406073776570368.000000
hardPoint = -91.150002 1.015596 673.400024
wheelDirection = -0.000000 -1.000000 0.000000
suspensionLength = 0.000000
hardPoint = -92.849998 1.115596 676.599976
wheelDirection = -0.000000 -1.000000 0.000000
suspensionLength = 0.000000
hardPoint = -91.150002 1.115596 676.599976
wheelDirection = -0.000000 -1.000000 0.000000
suspensionLength = 0.000000
hardPoint = -92.849998 1.015596 673.400024
wheelDirection = -0.000000 -1.000000 0.000000
suspensionLength = -663963820797107406073776570368.000000
```
tried adding `controller.resetSuspension()` in `PhysicsVehicle.createVehicle()` but that moved the wheels to the vehicle's center

**Comment by stephengold** [comment]
Reverting 08b70b6045a8ef00b7f1d17893e461763c8a1370 appears to solve the issue. Not sure why, though.

**Comment by stephengold** [comment]
tried adding `wheel.setSuspensionLength(0f);` in the loop of `PhysicsVehicle.createVehicle()` --- worked the 1st time but after that it moved the wheels to the vehicle's center

**Comment by stephengold** [comment]
There was another change to Minie that was interfering with the test. Reverted that at 3b16784ed2ae2c7f817cad2ed2aaa9c0c10b1527 and then committed the fix at a505453536a764c2adaf55862f88ca7d850529c3. I believe the issue is solved as of Minie v4.0.0.
